node/dataSources/checkout.ts

```diff
@@ -93,6 +101,14 @@ export class CheckoutDataSource extends RESTDataSource<ServiceContext> {
     simulation,
   )
 
+  public simulation = (simulation: SimulationData, query: any) => {
+    const sanitizedQuery = stringify(query).trim()
```

Comment:
@firstdoit Does this work as intended?

Reply:
I guess so, why did you ask? Something broke? We just don't want to have any spaces when sending the request to the catalog proxy, so we dont cache `Linda` and `Linda ` separately in the pod.